CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ All notable changes to this project will be documented in this file.
 ### Fixed
 
 -   #1587 : Fix unnecessarily long file names generated by `epyccel`.
+-   #1576 : Corrected Destructor Invocation for Proper Cleanup.
+-   #1576 : Remove inline class method definition.
```

Comment:
How about issue #1444?

Reply:
I don't think issue #1444 explains anything about these problems. In my opinion it is more useful to link to this PR than that issue

Reply:
The title of this PR is "Test functions which pass and return classes" but we do not mention that...

Reply:
I usually ask people to not mention tests in the CHANGELOG. The CHANGELOG is designed for users. If we include tests in this then we will have a line in every PR about tests added

Reply:
We could rename the PR:
> Ensure support for functions which pass and return classes